<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_WC_AddProcDialogClass</class>
 <widget class="QDialog" name="Qt_WC_AddProcDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_WC_AddProcDialog</string>
  </property>
  <widget class="QLineEdit" name="le_procDir">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>122</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_chkDelay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>199</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>123</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>프로그램 경로 &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>지연시간 ( 딜레이 ) &gt; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>중복방지 &gt; </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_chkDup">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_chkApp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>59</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>APP</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_chkWeb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>59</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>WEB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>타입 &gt; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_url">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>이동시킬 URL &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_procSearch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid #000000;
	background-color : #ffffff;
}

QPushButton:hover{
	border:1px solid #000000;
	background-color : #f0f0f0;
}

QPushButton:pressed{
	border:1px solid #060606;
	background-color : #f4f4f4;
}
</string>
   </property>
   <property name="text">
    <string>찾기...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_procName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>프로그램 명 &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>프로그램 선택 &gt; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_AddProcOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid #000000;
	background-color : #ffffff;
}

QPushButton:hover{
	border:1px solid #000000;
	background-color : #f0f0f0;
}

QPushButton:pressed{
	border:1px solid #060606;
	background-color : #f4f4f4;
}
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_AddProcCancle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid #000000;
	background-color : #ffffff;
}

QPushButton:hover{
	border:1px solid #000000;
	background-color : #f0f0f0;
}

QPushButton:pressed{
	border:1px solid #060606;
	background-color : #f4f4f4;
}
</string>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
